tasks/scrapeData/rateLocations.js

```diff
@@ -91,4 +94,40 @@ function calculateRating(info) {
   return rating / possible;
 }
 
-export default calculateRating;
+const rateLocations = args => {
+  const { locations } = args;
+
+  const sourcesByURL = {};
+  for (const location of locations) {
+    const sourceObj = { ...location._scraperDefinition };
+    for (const prop of sourceProps) {
+      if (location[prop] !== undefined) {
+        sourceObj[prop] = location[prop];
+      }
+    }
+    for (const prop in sourceObj) {
+      if (prop[0] === '_') {
```

Comment:
also repetitious with transform.js remove private thing

Reply:
I'm going to keep this to avoid changing too much at once. We can remove this in a future PR (which will happen soon I promise!)

Reply:
#212 to track